### DISCUSSION TITLE
How do you increase font size or dot size?

### DISCUSSION THREAD

**Discussion by lwtan90** [GMOD/jbrowse-components#4640] [General]
Hi,

The current font size is too small, and not suitable to show during presentation. How do you increase both font size and dot size for the xyplot?

Thank you.

Wil

**Comment by lwtan90** [comment]
I am using the interface, not the command line version.

**Reply by cmdcolin** [reply]
can you post a screenshot with circles around the elements you want to enlarge?

**Comment by lwtan90** [comment]
<img width="1450" alt="Screen Shot 2024-11-06 at 12 56 05 PM" src="https://github.com/user-attachments/assets/e6b98faa-ede0-4d31-966f-5cf89db24ecf">

Example, those that I have circled. Thank you!

I am wondering if there is a setting tab somewhere that will alter the font or resolution globally.
Thank you.

**Reply by cmdcolin** [reply]
The dot size on the coverage can be increased by going to the vertical ... and selectign "no fill w/ emphasis"

it's kind hacky that it isn't an arbitrary sized unit to scale to 

screenshot


![image](https://github.com/user-attachments/assets/75536402-fc63-4ea4-852a-23eb21927dd7)


the adjustment of fontsize is not well supported yet. you can adjust some fonts in the UI via the theme (https://jbrowse.org/jb2/docs/config_guides/theme/#sizing) but the two places you circled in the UI actually hardcode the fonts. we can try to make it responsive to that theme variable though in a future version

**Reply by cmdcolin** [reply]
two other alternatives

a) making an svg export and editing fonts in e,g, inkscape
b) using browser zoom. on web, you can hold ctrl and hit the + button. on desktop, the zoom is a little hidden maybe, but you can hit ctrl+shift+i (or cmd+shift+i) and then put in the devtools console "       document.body.style.zoom = "150%";"

![image](https://github.com/user-attachments/assets/9cde2a09-649d-458f-a8f2-f206390fa1eb)
